src/main/java/utility/PropertiesLoader.java

```diff
@@ -0,0 +1,79 @@
+package utility;
+
+import com.google.common.base.Strings;
+import lombok.SneakyThrows;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Properties;
+
+import static java.util.Objects.*;
+
+public class PropertiesLoader {
+
+    private static final String PROPERTIES_FILE = "/properties/application.properties";
+    private static final String TEST_DATA_FILE = "/properties/testdata.properties";
+    private static final Properties PROPERTIES = getPropertiesInstance(PROPERTIES_FILE);
```

Comment:
Consider initialize this in static block, not in field.

Reply:
may I ask for some references because I am not sure.